doc/roadmap.md

```diff
@@ -26,40 +26,52 @@ The following list of features are planned to be addressed in the short term, an
+-   Remove plugins structure (bidirectional plugin) ([#1413](https://github.com/telefonicaid/iotagent-node-lib/issues/1413))
+-   Add init and improve log traces ([#1452](https://github.com/telefonicaid/iotagent-node-lib/issues/1452))
+-   Remove InMemory registry for Devices and Groups ([#1429](https://github.com/telefonicaid/iotagent-node-lib/issues/1429))
 
 ### Medium term
 
 The following list of features are planned to be addressed in the medium term, typically within the subsequent
 release(s) generated in the next 9 months after the next planned release:
 
--   Accept JEXL Expressions for entity name in autoprovisioned devices (#1145)
--   Refactor entities-NGSI-v2.js module (#1166)
+-   Allow to add metadata to attributes from measures ([#1453](https://github.com/telefonicaid/iotagent-node-lib/issues/1453))
+-   Cache support ([#1467](https://github.com/telefonicaid/iotagent-node-lib/issues/1467))
+-   MQTT per group advanced configuration (MQTT broker, topics) ([#1454](https://github.com/telefonicaid/iotagent-node-lib/issues/1454))
+-   Subscription based commands ([#1455](https://github.com/telefonicaid/iotagent-node-lib/issues/1455))
+-   Remove registration support ([#1456](https://github.com/telefonicaid/iotagent-node-lib/issues/1456))
 
 ### Long term
 
 The following list of features are proposals regarding the longer-term evolution of the product even though the
 development of these features has not yet been scheduled for a release in the near future. Please feel free to contact
 us if you wish to get involved in the implementation or influence the roadmap:
 
+-   Use the lightweight ingestion mechanism for connection oriented updates implemented in Context Broker (testing pending) ([#1457](https://github.com/telefonicaid/iotagent-node-lib/issues/1457))
+-   Add support to other transport protocols (BacNET, Modbus, etc) ([#1458](https://github.com/telefonicaid/iotagent-node-lib/issues/1458))
+-   Dynamic attribute generation (based on values array) ([#1459](https://github.com/telefonicaid/iotagent-node-lib/issues/1459))
 -   Incremental introduccion of ECMAScript6 syntax (previous analysis of which sub-set of interesting aspect we want to
     take)
--   Use the lightweight ingestion mechanism for connection oriented updates implemented in Context Broker
--   Add support to other transport protocols (BacNET, Modbus, etc)
 
 ### Features already completed
 
 The following list contains all features that were in the roadmap and have already been implemented.
 
+-   Refactor entities-NGSI-v2.js module ([#1166](https://github.com/telefonicaid/iotagent-node-lib/issues/1166)) ([3.0.0](https://github.com/telefonicaid/iotagent-node-lib/releases/tag/3.0.0))
+-   Accept JEXL Expressions for entity name in autoprovisioned devices (entityNameExp) ([#1145](https://github.com/telefonicaid/iotagent-node-lib/issues/1145)) ([2.24.0](https://github.com/telefonicaid/iotagent-node-lib/releases/tag/2.24.0))
```

Comment:
I don't find #1145 mentioned in https://github.com/telefonicaid/iotagent-node-lib/releases/tag/2.24.0. However, it's mentioned in https://github.com/telefonicaid/iotagent-node-lib/releases/tag/2.22.0

Is this link ok?

Reply:
As far as I can see, merge commit of related PR is taged as 2.22.0:
https://github.com/telefonicaid/iotagent-node-lib/commit/ff6bfe9689c1423c9d78067eef5f51720975fa67

Reply:
> As far as I can see, merge commit of related PR is taged as 2.22.0: [ff6bfe9](https://github.com/telefonicaid/iotagent-node-lib/commit/ff6bfe9689c1423c9d78067eef5f51720975fa67)

That's confirm my point:

> However, it's mentioned in https://github.com/telefonicaid/iotagent-node-lib/releases/tag/2.22.0

So:

```suggestion
-   Accept JEXL Expressions for entity name in autoprovisioned devices (entityNameExp) ([#1145](https://github.com/telefonicaid/iotagent-node-lib/issues/1145)) ([2.22.0](https://github.com/telefonicaid/iotagent-node-lib/releases/tag/2.22.0))
```

It's correct?